src/main/java/net/sf/jabref/importer/CustomImporter.java

```diff
@@ -117,6 +117,10 @@ public URL getBasePathUrl() throws MalformedURLException {
 
     @Override
     public boolean equals(Object o) {
+        if (o == null) {
+            return false;
+        }
+
         boolean equalName = this.getName().equals(((CustomImporter) o).getName());
```

Comment:
That is correct. Implementations of equals should always follow a very narrow contract. @oscargus: If you are interested, this is excellently described in *Effective Java 2nd Edition*, Item 8: "Obey the general contract when overriding equals" (I won't link a pdf version here, but I am sure you can find one if you desire).

Most IDEs can auto-generate you a good implementation of equals these days.